packages/cli/src/util/dev/server.ts

```diff
@@ -135,9 +135,18 @@ export default class DevServer {
   public proxy: httpProxy;
   public envConfigs: EnvConfigs;
   public files: BuilderInputs;
-  public address: string;
-  public devCacheDir: string;
 
+  private address: URL | undefined;
+  public get addressUrl(): URL {
```

Comment:
Let call this `addressURL` with uppercase URL

Reply:
The casing is inconsistent in this repo. It seems that for this specific case, `somethingUrl` (376 results) is more common than `somethingURL` (50 results).

Is there a defined team convention on casing that we're trying to stick to? I'd be happy to always use one or the other if there is.

Reply:
I'm basing it on the web standards

- URL
- URLSearchParams
- URLPattern

Reply:
That being said, I prefer removing it in favor of https://github.com/vercel/vercel/pull/8538/files#r967401450

Reply:
Updated casing: [`735b0b6` (#8538)](https://github.com/vercel/vercel/pull/8538/commits/735b0b67cdef3d838e75f6328537d3f215d154d0)